docs/application/native/guides/multimedia/sound-manager.md

```diff
@@ -152,11 +153,16 @@ To query sound device information:
       while ((_ret = sound_manager_get_next_device(list, &device)) == SOUND_MANAGER_ERROR_NONE) {
           ret = sound_manager_get_device_type(device, &type);
 
-          if (type == SOUND_DEVICE_BLUETOOTH)
-              /* Sound device type is Bluetooth, handle accordingly */
+          if (type == SOUND_DEVICE_BLUETOOTH_MEDIA)
+              /* Sound device type is Bluetooth Media, handle accordingly */
```

Comment:
/* Sound device type is Bluetooth Media, handle accordingly */
What  we mean by "handle accordingly"? Please clarify.

Reply:
Honestly I don't know who put this phrase - handle accordingly - in all if-else statements.
Rather I would like to remove this pharse.

Reply:
I agree with you.